pinotdb/db.py

```diff
@@ -348,6 +352,7 @@ def __init__(
     @check_closed
     def close(self):
         """Close the cursor."""
+        # TODO: Check if the session actually exists, first.
```

Comment:
can we just do the check here instead of adding a TODO.

Reply:
OK, I can add it to this PR. I wanted to avoid making the PR too big, but it's fine for me to fix that now.

Reply:
Fixed!